lib/js/components/Drawer/DrawerSection/DrawerSection.stories.ts

```diff
@@ -12,25 +18,97 @@ const StoryTemplate: StoryFn<typeof DrawerSection> = (args) => ({
 	setup() {
 		return { ...args };
 	},
+	data() {
+		return {
+			ICONS: Object.freeze(ICONS),
+		};
+	},
+	methods: {
+		infoClicked() {
+			console.log('info clicked');
+		},
+	},
 	template: `
 		<drawer-section
-			:has-divider="hasDivider"
-			:is-expandable="isExpandable"
+			:size="size"
 			:title="title"
+			:supporting-text="supportingText"
+			:eyebrow="eyebrow"
+			:icon-left="ICONS[iconLeft]"
+			:icon-left-color="iconLeftColor"
+			:icon-right="ICONS[iconRight]"
+			:icon-right-color="iconRightColor"
+			:info="info"
+			:divider="divider"
+			:is-expandable="isExpandable"
+			:is-expanded="isExpanded"
+			:hide-slot-when-collapsed="hideSlotWhenCollapsed"
+			:mobile-layout="mobileLayout"
+			@infoClick="infoClicked"
 		>
-		Default slot
-		<template #uncollapsible>Uncollapsible slot</template>
+			Default slot
+			<template #uncollapsible>Uncollapsible slot</template>
 		</drawer-section>`,
 });
 
 export const Interactive = StoryTemplate.bind({});
 
 const args = {
-	hasDivider: true,
+	size: SECTION_HEADER_SIZES.XX_SMALL,
```

Comment:
I wonder if this should be accessible. In figma we have that xxs size should be used 
https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS-Components?type=design&node-id=6785-111048&mode=dev

Reply:
It's listed as one of prop in miro:
<img width="467" alt="image" src="https://github.com/bethinkpl/design-system/assets/2042649/f2e22a97-2eeb-4070-8e4e-795f7734b70a">

Reply:
Yes, and it is inconsistent with Figma